WalletWasabi/Helpers/PowerSaving/WindowsPowerAvailabilityTask.cs

```diff
@@ -158,10 +161,18 @@ public Task StopAsync()
 
 			if (!isDone)
 			{
-				Logger.LogTrace("Clear the power request.");
-				PowerClearRequest(_request, RequestType);
+				Logger.LogTrace("Clear the power requests.");
+				PowerClearRequest(_hibernationRequest, PowerRequestType.PowerRequestExecutionRequired);
+
+				if (CloseHandle(_hibernationRequest) == 0)
+				{
+					// This should never happen.
+					Logger.LogError($"Failed to close handle (last Win32 error: {Marshal.GetLastWin32Error()}).");
+				}
 
-				if (CloseHandle(_request) != 0)
+				PowerClearRequest(_sleepRequest, PowerRequestType.PowerRequestDisplayRequired);
+
+				if (CloseHandle(_sleepRequest) == 0)
```

Comment:
According to this: https://docs.microsoft.com/en-us/windows/win32/api/handleapi/nf-handleapi-closehandle

> If the function fails, the return value is zero.

> If the function succeeds, the return value is nonzero.

Which feels like its in reverse, but that's how its implemented.